src/protocols/secure_channel/tests/TestCASESession.cpp

```diff
@@ -435,6 +453,56 @@ void CASE_SecurePairingSerializeTest(nlTestSuite * inSuite, void * inContext)
     chip::Platform::Delete(testPairingSession2);
 }
 
+void CASE_DestinationIDGenerationTest(nlTestSuite * inSuite, void * inContext)
```

Comment:
This does not test the version present in the spec. How was this test case generated? Why not test against the test vectors from the spec?